[{"user_id": 35238, "stars": [], "topic_id": 38996, "date_created": 1307760520.2321579, "message": "It appears that only celeryd is launched automatically, right? Not like that's a problem\u2014I guess, it's okay to put celerybeat and celerycam to [cron] section.", "group_id": 2873, "id": 1368307}, {"user_id": 35238, "stars": [], "topic_id": 38996, "date_created": 1307761417.8139441, "message": "I'm using django-celery, settings are placed into django's settings module.", "group_id": 2873, "id": 1368346}, {"user_id": 35238, "stars": [], "topic_id": 38996, "date_created": 1307760931.849009, "message": "But there's also seems to be a problem with celeryd: I can't get settings like CELERYD_OPTS or CELERYD to work. So there's no way to, e.g., launch celeryd with `-E` to enable events so that celerycam would show in admin what's going on, or `-l info` to change logging level. (Or am I missing something?)", "group_id": 2873, "id": 1368331}, {"user_id": 8740, "stars": [], "topic_id": 38996, "date_created": 1307789059.141711, "message": "except there's no PID file, as this isn't a normal system you're running on :)", "group_id": 2873, "id": 1369411}, {"user_id": 8740, "stars": [], "topic_id": 38996, "date_created": 1307789079.923281, "message": "and yes, not having a logging level as a configuration option is a little odd", "group_id": 2873, "id": 1369413}, {"user_id": 8740, "stars": [], "topic_id": 38996, "date_created": 1307784426.1198981, "message": "no, there's no way to change the command-line options currently - we'd need to add options for that", "group_id": 2873, "id": 1369291}, {"user_id": 8740, "stars": [], "topic_id": 38996, "date_created": 1307784453.8242259, "message": "also, I advise against running celerybeat/cam under cron, as they won't get killed when the next one starts, so you'll get e.g repeat events", "group_id": 2873, "id": 1369292}, {"user_id": 35238, "stars": [], "topic_id": 38996, "date_created": 1307787361.2216351, "message": "celerybeat: I though about adding to cron some sort of shell script, which would check if it's already running via pid file. ", "group_id": 2873, "id": 1369364}, {"user_id": 35238, "stars": [], "topic_id": 38996, "date_created": 1307787272.7571959, "message": "Turned out that CELERYD_OPTS isn't supposed to have any effect being placed in settings.py (I misread celery docs)... I can't beleive there's no way to, e.g., set logging level without changing command-line options.", "group_id": 2873, "id": 1369360}, {"user_id": 8740, "stars": [], "topic_id": 38996, "date_created": 1307789098.3869081, "message": "I'll take a look at it once we're on the new backend to make sure we have the relevant options exposed", "group_id": 2873, "id": 1369417}, {"user_id": 25736, "stars": [], "topic_id": 38996, "date_created": 1307861456.240468, "message": "@rain_zuko what I did was run the celerybeat tasks as cron tasks - so they are ran anyway but also ready for when celerybeat is supported.", "group_id": 2873, "id": 1373316}, {"user_id": 35238, "stars": [], "topic_id": 38996, "date_created": 1307882325.824815, "message": "@liorsion, thanks for the advice, I guess I'll go with it for now.", "group_id": 2873, "id": 1374144}, {"user_id": 35238, "stars": [], "topic_id": 38996, "date_created": 1307883181.3348939, "message": "@andrewgodwin, epio django celerybeat -- --pidfile=../data/celerybeat.pid does the trick. Hovewer, turned out I can't use pidfile to kill the task anyway, kill `cat data/celerybeat.pid` says \"No such process\".", "group_id": 2873, "id": 1374238}, {"user_id": 8740, "stars": [], "topic_id": 38996, "date_created": 1307890893.4154651, "message": "@rain_zuko Yes, because the commands are probably running on different servers. Your app doesn't live just on one - that's what I meant by \"it's not a normal system\" :)", "group_id": 2873, "id": 1374509}, {"user_id": 35238, "stars": [], "topic_id": 38996, "date_created": 1307926208.5461891, "message": "Oh. I see now, thanks for the explanation =)", "group_id": 2873, "id": 1376672}]